### PR TITLE
enable kube e2e tests

### PR DESCRIPTION
**Pull request by jrhizor**
Reverts the testing component of https://github.com/airbytehq/airbyte/pull/2315

My hope is that these are more stable after our updates + the new EC2 test runners. 

Keeping as a draft until these are passing + go through a few rounds.

### CODE REVIEW

**Comment by jrhizor** [comment]
actually I still need to update this to use the new runner

**Comment by jrhizor** [comment]
I _think_ this is all working now except for normalization which was missing an entrypoint. I don't think it'll build until we push that image. Testing locally.

**Comment by jrhizor** [comment]
These are the two interesting commits so far:
https://github.com/airbytehq/airbyte/pull/3866/commits/8f4d8267c71bfdb5428f7efebbb40d2226a76b7a
https://github.com/airbytehq/airbyte/pull/3866/commits/928762a137ff88d9a63808f1a4e2148824bede51

File: tools/bin/acceptance_test_kube.sh [hunk 1]
```diff
@@ -14,7 +14,13 @@ kubectl apply -k kube/overlays/dev
 kubectl wait --for=condition=Available deployment/airbyte-server --timeout=300s || (kubectl describe pods && exit 1)
 kubectl wait --for=condition=Available deployment/airbyte-scheduler --timeout=300s || (kubectl describe pods && exit 1)
 
-sleep 20s
+sleep 120s
```
Comment (davinchia) [inline tools/bin/acceptance_test_kube.sh:17]: nit: comment here to explain

Comment (jrhizor) [reply tools/bin/acceptance_test_kube.sh:17]: not a nit

File: airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java [hunk 2]
```diff
@@ -323,10 +328,11 @@ public KubePodProcess(KubernetesClient client,
     LOGGER.info("Creating pod...");
     this.podDefinition = client.pods().inNamespace(namespace).createOrReplace(pod);
     waitForInitPodToRun(client, podDefinition);
-    if (copyFiles) {
-      LOGGER.info("Copying files...");
-      copyFilesToKubeConfigVolume(client, podName, namespace, files);
-    }
+
+    LOGGER.info("Copying files...");
```
Comment (davinchia) [inline airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java:332]: nit: comment on why this is the case

File: airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java [hunk 3]
```diff
@@ -75,6 +75,14 @@ public Process create(String jobId,
 
       return new KubePodProcess(
           kubeClient,
+          port -> {
```
Comment (davinchia) [inline airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java:78]:
```
var portReleaser = port -> {
  if (!ports.contains(port)) {
              ports.add(port);
              LOGGER.info("Port consumer releasing: " + port);
            } else {
              LOGGER.info("Port consumer skipping releasing: " + port);
            }
}

new KubePodProcess(
  kubeClient,
  portReleaser,
.....);
```

**Review by davinchia — APPROVED** [review]
Looks great. Feel free to merge after comments and nits.

**Pull request merged**